react/features/toolbar/components/secondaryToolbarHandlers.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+
+import UIEvents from '../../../../service/UI/UIEvents';
+
+declare var APP: Object;
+declare var JitsiMeetJS: Object;
+declare var config: Object;
+
+/**
+ * Shows SIP number dialog.
+ *
+ * @returns {void}
+ */
+function _showSipNumberInput() {
+    const defaultNumber = config.defaultSipNumber || '';
+    const titleKey = 'dialog.sipMsg';
+    const msgString = `
+            <input class="input-control"
+                   name="sipNumber" type="text"
+                   value="${defaultNumber}" autofocus>`;
+
+    APP.UI.messageHandler.openTwoButtonDialog({
+        titleKey,
+        msgString,
+        leftButtonKey: 'dialog.Dial',
+
+        // eslint-disable-next-line max-params
+        submitFunction: (event, value, message, formValues) => {
+            const { sipNumber } = formValues;
+
+            if (value && sipNumber) {
+                APP.UI.emitEvent(UIEvents.SIP_DIAL, sipNumber);
+            }
+        },
+        focus: ':input:first'
+    });
+}
+
+export default {
```

Comment:
I and Yana (as far as I understand) find the separation between `primaryToolbarHandlers` and `secondaryToolbarHandlers` inappropriate. The words `primary` and `secondary` refer to UI/presentation while these are handlers i.e. (business) logic. Please remove the separation between `primaryToolbarHandlers` and `secondaryToolbarHandlers`.